numpy/core/src/multiarray/shape.c

```diff
@@ -87,7 +87,14 @@ PyArray_Resize(PyArrayObject *self, PyArray_Dims *newshape, int refcheck,
         }
 
         if (refcheck) {
+#ifdef PYPY_VERSION
+            PyErr_SetString(PyExc_ValueError,
+                    "cannot resize an array on PyPy."\
+                    "  Use the resize function");
```

Comment:
I was just copying the error message a few lines later, so if I am to change this I should update that one as well (both for slashes and ``refcheck=False``)? I would prefer to leave it for now...